src/main/java/uk/gov/hmcts/reform/sscs/service/CitizenLoginService.java

```diff
@@ -181,17 +178,9 @@ public void findAndUpdateCaseLastLoggedIntoMya(IdamTokens citizenIdamTokens, Str
     }
 
     private void updateCaseWithLastLoggedIntoMya(String email, SscsCaseDetails caseByAppealNumber) {
-        if (citizenLogicServiceV2Enabled) {
-            log.info("Updating case with last logged in MYA using V2, case id: {}, matching email: {}", caseByAppealNumber.getId(), email);
-            updateCcdCaseService.updateCaseV2(caseByAppealNumber.getId(), EventType.UPDATE_CASE_ONLY.getCcdType(), "SSCS - update last logged in MYA",
-                    UPDATED_SSCS, idamService.getIdamTokens(), sscsCaseDetails -> updateSubscriptionWithLastLoggedIntoMya(sscsCaseDetails, email));
-        } else {
-            log.info("Updating case with last logged in MYA using V1, case id: {}, matching email: {}", caseByAppealNumber.getId(), email);
-            updateSubscriptionWithLastLoggedIntoMya(caseByAppealNumber, email);
-            ccdService.updateCase(caseByAppealNumber.getData(), caseByAppealNumber.getId(), EventType.UPDATE_CASE_ONLY.getCcdType(),
-                    "SSCS - update last logged in MYA", UPDATED_SSCS, idamService.getIdamTokens()
-            );
-        }
+        log.info("Updating case with last logged in MYA using V2, case id: {}, matching email: {}", caseByAppealNumber.getId(), email);
```

Comment:
Doesn't email id fall under GDPR?

Reply:
probably... but its an existing issue (even in V1) so I will let the CI team know

Reply:
raised with CI team